howtos/range_queries.rst

```diff
@@ -142,11 +142,7 @@ prefixes.
 If you get the rules wrong, the QueryParser will raise a
 `QueryParserError`, which in production code you could catch and
 either signal to the user or perhaps try the query again without the
-`RangeProcessor` that tripped up:
-
-.. xapianrunexample:: search_ranges
-    :args: db 1000mm..
-    :shouldfail: 1
+`RangeProcessor` that tripped up.
```

Comment:
Not opposed to it at all — what we should probably do is to have a discussion as we build the initial example of the types of exception that you most have to deal with. We can start with something specific in this case so we emit something standard and useful, and then go back and look at the rest later.